cmd/krew/cmd/search.go

```diff
@@ -118,6 +113,32 @@ Examples:
 	PreRunE: checkIndex,
 }
 
+func searchByNameAndDesc(keyword string, nameData, descData []string) []string {
+	var searchResults []string
+	if len(keyword) > 0 {
```

Comment:
Looks like you can make this easier to read this method by checking the ==0 first and returning, then moving this this block outside the if.